src/libsync/syncfileitem.h

```diff
@@ -94,6 +94,7 @@ class OWNCLOUDSYNC_EXPORT SyncFileItem
         Excluded
     };
     Q_ENUM(Status)
+    static constexpr int StatusCount = Excluded + 1;
```

Comment:
I prefer to have the count in the enum it self. Yes that means we would need to touch the switch statements where the status is used. This would also allow us to drop cases where default: is define and ensure we handle all cases.